File: python/src/euler/problem005.py
def smallest_value_without_reminder(min, max):
    smallest_number = None

    counter = max
    while smallest_number is None:
        with_reminders = False
        for i in range(min, max):
            if counter % i != 0:
                with_reminders = True
                break

        if with_reminders:
            counter += 1
            continue

        smallest_number = counter

    return smallest_number

File: java/src/main/java/euler/Problem005.java
package euler;

public class Problem005 {

    public long smallestNumberDividedWithoutAnyReminder(final int start, final int end) {
        Long smallestNumber = null;

        long counter = end;
        while (smallestNumber == null) {
            boolean withReminders = false;
            for (int i = start; i < end; i++) {
                if (counter % i != 0) {
                    withReminders = true;
                    break;
                }
            }

            if (withReminders) {
                counter++;
                continue;
            }

            smallestNumber = counter;
        }

        return smallestNumber;
    }
}
